jenkins/pipeline/gxf-nightly-build.jenkinsfile

```diff
@@ -0,0 +1,220 @@
+// Pipeline script for the OSGP Nightly Build job in Jenkins
+/* groovylint-disable CompileStatic, DuplicateStringLiteral, LineLength, NestedBlockDepth, NoDef, VariableTypeRequired */
+
+def stream = 'osgp'
+def servername = stream + '-at-' + env.BUILD_NUMBER
+def playbook = stream + '-at.yml'
+def branchReleaseRepo = 'feature/OC-712-run-automatic-tests-in-containerized-jenkins'
```

Comment:
Will have to be updated depending on this pr: https://github.com/SmartSocietyServices/release/pull/235#pullrequestreview-571268096